feather/plugin-host/src/host_calls/entity.rs

```diff
@@ -17,12 +17,11 @@ pub fn entity_send_message(
     message_ptr: PluginPtr<u8>,
     message_len: u32,
 ) -> anyhow::Result<()> {
-    let message = cx.read_string(message_ptr, message_len)?;
+    let message = cx.read_json(message_ptr, message_len)?;
```

Comment:
We are blindly trusting plugins to send valid text messages as json without confirming?

Reply:
I think this is fine for now, but shows that the current `Text` api is a bit lacking...

Reply:
The text component is serialized with serde_json in quill/api